git-cliff-core/Cargo.toml

```diff
@@ -78,6 +78,7 @@ dyn-clone = "1.0.17"
 urlencoding = "2.1.3"
 cacache = { version = "13.0.0", features = ["mmap"], default-features = false }
 time = "0.3.36"
+test-tag = "0.1.4"
```

Comment:
Why this is needed?

Reply:
It's not, just a leftover of a previous attempt, thanks.